app/views/govuk_publishing_components/components/_glance_metric.html.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  add_gem_component_stylesheet("glance-metric")
+
+  name ||= false
+  figure ||= nil
+  context ||= ""
+  measurement_display_label ||= ""
+  measurement_explicit_label ||= ""
+%>
+<% if name %>
+  <div class="gem-c-glance-metric govuk-body">
```

Comment:
For future proofing, could you add the component wrapper helper to this component? Otherwise it creates more work for our team.

For example see this commit: https://github.com/alphagov/govuk_publishing_components/pull/4287/commits/e2a09b40bff246066330225226d781a364fd71af (However you'd only need to use `add_class` for yours, as your parent `<div>` only uses CSS classes.)